src/ldp/http/SimpleResponseWriter.ts

```diff
@@ -40,6 +39,7 @@ export class SimpleResponseWriter extends ResponseWriter {
       if (input.error instanceof HttpError) {
```

Comment:
If you use `instanceof` in an `if` block, TypeScript is smart enough to realize that within the scope of that block that variable is an instance of what was checked.

Reply:
Yes, but [below](https://github.com/solid/community-server/pull/37/files/fca08a3d9531b006654bb648e68dc647c95e726a#diff-71ead7cf50485323a37e150346bd7696L44) you are accessing `input.error` outside of the `if` block?

Reply:
Ah you mean that there is no specific check to see if `error` is not undefined but it's still accessed? That's not something TypeScript complains about with the current project settings.

Reply:
> That's not something TypeScript complains about

Should we reconsider?

Reply:
Not sure if I've mentioned this before, but I'd strongly encourage setting `strict` to `true`. Easier to do at the start of a project than later on :-)

Reply:
strict is set to true, it's `strictNullChecks` that is required (unless that is what you meant). I remember having that setting initially but disabling it since it conflicted with some things. I don't fully remember why though, could open an issue to re-enable it and look into it again.

Reply:
AFAICS only `alwaysStrict` is set to true, which is different than `strict` (the latter does more).

Reply:
Ah I missed that specific setting. Will open an issue to enable it then.